hugo/layouts/index.html

```diff
@@ -27,9 +27,7 @@
         {{ partial "home/blog.html" . }}
     {{ end }}
 
-    {{ partial "footer/slack.html" . }}
-    {{ partial "footer/contact-us.html" . }}
-    {{ partial "footer/sitemap.html" . }}
+    {{ partial "footer.html" . }}
```

Comment:
Not all pages will have all footer component;
a) separate each one again,
b) use a parametric `footer.html` partial to be configured from the other lages